resources/istio/INSTALL.v1.14.md

```diff
@@ -0,0 +1,3 @@
+# Prometheus metrics configuration
+
+Istio already expose by default all its metrics, so is not necessary make any configuration in order to enable those metrics.
```

Comment:
@jaimeyh Please fix this in 'staging'

"Istio already exposes all its metrics in Promethes format by default, so it's not necessary to modify any configuration in order to enable those metrics."

Reply:
Done